serpentTools/parsers/bumat.py

```diff
@@ -12,4 +12,101 @@ class BumatReader(MaterialReader):
     filePath: str
         path to the depletion file
     """
-    pass
+
+    def __init__(self, 
```

Comment:
`__init__` block is syntactically incomplete

Reply:
right, I'm going to include more once we figure out what kind of data structure these isotope densities should be enclosed within.